Documentation/wip/support-tool.md

```diff
@@ -0,0 +1,62 @@
+# Fission Support Tool
+Fission now has rich functionality supported by multiple services, however, it brings the complexity of troubleshooting.
+This proposal tends to give a picture of fission support tool that can help both user and developer to locate the problem in short time.
+To achieve this, the support tool will dump related kubernetes objects, fission resources and pod logs from the given cluster.
+
+# Functionality
+
+## Environment Information Collection
+
+Before troubleshooting, some of the basic information is needed to give others an overview of kubernetes/fission user test with so that we can locate the problem in short time.
+
+* Fission version
+    * Client/Server version
```

Comment:
You might want to dump some CLI/host machine information here too, especially as they are quite often a culprit:
- arch/os
- resource usage (disk space free, memory free...)

Of course we always add these sources of info and more later on once we discover we need them.